### ISSUE TITLE
move svn log hint message for log notes printed by install.sh to end of output

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1690]
_Imported from trac ticket 1690.
Created by **rneville**
Opened in trac: Jun 4, 2012, 8:55:44 AM
Last modified in trac: Jun 5, 2012, 8:25:31 AM_
***
_by rneville:_
so we don't have to scroll up past mxml compilation feedback to see it

**Comment by kbower** [comment]
_**kbower** trac comment on Jun 4, 2012, 11:11:03 AM:_

Yeah, the issue is that it isn't `install.sh` that outputs that line or has any knowledge of it.  It is `paster setup-app`.  Can be worked out somehow, but in no solid way.  If we "pass" that line to `install.sh` then on some other failure, like flex compilation, then when rerun, `install.sh` won't output any line.

When I get a minute, I'll add this line to the end of `install.sh` iff it succeeds:
```
ls -1t $RDIR/src/appserver/setup.*.log | head -1 | xargs grep "^svn log"
```

**Comment by kbower** [comment]
_**kbower** trac comment on Jun 4, 2012, 11:14:20 AM:_

Better:
```
ls -1t $RDIR/src/appserver/setup.*.log 2>/dev/null | head -1 | xargs grep "^svn log"
```

**Comment by kbower** [comment]
_**kbower** trac comment on Jun 5, 2012, 8:25:31 AM:_

(Changeset [[2724]](../commit/50aa5a902838409c6d5eef2f3b545fa178646a86))
 * move svn log hint message for log notes printed by install.sh to end of output (closes #1690)

**Comment by kbower** [comment]
_**kbower** trac status change on Jun 5, 2012, 8:25:31 AM:_ **closed**
